lib/styles/components/_buttons.scss

```diff
@@ -3,48 +3,183 @@
 @import '../settings/spacings';
 @import '../settings/radiuses';
 @import '../settings/typography';
+@import '../settings/buttons';
 
-$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black, 0.35);
+$default-lighter-shadow: 0 4px 12px rgba($color-firefly-black-rgb, 0.35);
```

Comment:
firefly? Isn't it banned now?

Reply:
It is, I'm replacing it to `var(--raw-black-rgb)`